### ISSUE TITLE
Major publishing groups should agree to bioboxes standard.

### ISSUE THREAD

**Issue by michaelbarton** [bioboxes/rfc#47]
I think a large portion of the success of the MIAME standard was publishers
requiring submissions containing microarray data to include the required
information. Therefore I believe we should discuss this standard with
publishers as we move towards v1.0 and ask them to encourage editors/reviews to
follow it when reviewing software.

**Comment by michaelbarton** [comment]
I think @ScottBGI at Gigascience can provide with useful suggestions and
feedback on this process.

**Comment by ScottBGI** [comment]
Yes, let me know what I can do to help. We are already getting these types of submissions at GigaScience, so can use these test cases to work on best practices and fine tune and any standards. As we use open review as standard this will be transparent and can be referred back to in any developing documentation as well.

**Comment by michaelbarton** [comment]
Thanks for your response @ScottBGI. Could you share what sort of information
you would need to implement a community standard from the publisher's point of
view? This will help us shape the RFC as we start towards v1.0.

**Comment by ScottBGI** [comment]
Even though a depressing number of people will miss or ignore any guidelines, as you say making MIAME style minimal information checklists could help. For reviewers and editors to actually use it the more concise the better, defining essential/recommended information. Publishers would probably want flexibility on thresholds and licensing, although bioboxes/BOSC/OSI/etc. can express their own recommendations. Validation tools could also be useful too, as I think a few proteomics journals have done something similar, setting up pipelines with the EBI PRIDE database and made their curation and validation processes part of the review process.

**Comment by michaelbarton** [comment]
I spoke with @scottbgi, @pli888 and @bobbledavidson from @gigascience and they
provided excellent ideas and feedback on how we could work with publishers. I
will summarise the main points:

  * A short editorial could provide a call-to-arms for the community to help
    provide feedback on how the spec and how the containers could be used in
    bioinformatics. The aim of an editorial would be to highlight the benefits
    of the project, and encourage people to participate. An editorial limited
    to 1200 words with 10 references.

    My initial thought is to get this done after v0.9 as having a website and
    working validators will provide a more concrete example of the direction
    the project is moving in.

  * The BOSC conference would be an excellent place to spread the word about
    the project. The codefest would also be a great place to do perhaps a
    'dockerthon' about using containers and the benefits for bioinformatics.
    Having the above editorial completed before this date would be ideal also.
    Some members from gigascience should be at BOSC so it will also be a good
    time to discuss in person.

  * To encourate other publishers to adopt the standard, it should be as simple
    to use as possible and should have wide agreement and input from the
    bioinformatics community. If a proof of concept could be shown to add value
    to the publishing process, this would go a long way to helping the
    standard.

  * AppSoma and Galaxy are interested in using containers and so could provide
    technical feedback once we have a working version of the spec.

**Comment by michaelbarton** [comment]
Discussed in meeting #67. Adoption of bioboxes standards by journals will
become a greater priority once we have editor/reviewer instructions (#46) and
working validators (#35).
